var/spack/repos/builtin/packages/adlbx/package.py

```diff
@@ -26,6 +34,18 @@ def setup_environment(self, spack_env, run_env):
         spack_env.set('CXX', spec['mpi'].mpicxx)
         spack_env.set('CXXLD', spec['mpi'].mpicxx)
 
+    @when('@develop')
+    def configure_directory_helper(self):
+        return "lb/code"
+
+    @when('@0')
+    def configure_directory_helper(self):
+        return "."
+
+    @property
+    def configure_directory(self):
+        return self.configure_directory_helper()
```

Comment:
If you want to simplify this, you can do:
```python
@property
def configure_directory(self):
    if self.version == Version('develop'):
        return 'lb/code'
    else:
        return '.'
```
Then you don't need the helpers.

Reply:
Addressed in e3da0a81c .